src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -79,10 +84,12 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
   const roomEvents = useMemo(() => {
     if (!venueEvents) return [];
 
-    return venueEvents.filter(
-      (event) =>
-        event.room === room.title && isBefore(Date.now(), eventEndTime(event))
-    );
+    return venueEvents
+      .filter(
+        (event) =>
+          event.room === room.title && isBefore(Date.now(), eventEndTime(event))
+      )
+      .sort((a, b) => a.start_utc_seconds - b.start_utc_seconds);
```

Comment:
I believe we may already have a sorting function we can use here in one of the `utils/*` files.

Reply:
`venuePreview.tsx` line 163 appears to have an identical sort. I could pull this into a utils function, but could use some direction as to where to place it.

Reply:
replaced with `isEventLiveOrFuture` from `utils/events` 8fbc3f39505799cf9aba6470dac21321f5f7db56

Reply:
I've now replaced this sorting with the new function `sortEventsByStartUtcSeconds` in ef37ccd771f8e359c04cdb48cf443d54b6ae1ffc

Reply:
Sounds good. Can you please also replace the identical location with the function as well.

Reply:
great — replaced it in 76b8c3290427682c4d69e62fe42038740b3fd395

Reply:
Thanks :)